src/docfx/build/xref/XrefMap.cs

```diff
@@ -70,6 +71,16 @@ internal class XrefMap
             // xrefSpec.displayPropertyName -> xrefSpec.name -> uid
             string display = !string.IsNullOrEmpty(displayPropertyValue) ? displayPropertyValue : (!string.IsNullOrEmpty(name) ? name : uid);
             return (null, resolvedHref, display, referencedFile);
+
+            string RemoveHostnameIfSharingTheSameOne(string input)
```

Comment:
do we need consider the http or https case? same host name but different schema

Reply:
since the hostname contains `http(s)`, it should match?

Reply:
That'll make things a lot complicated, url comparison is itself very complex if you also consider port, www, etc. So we just right now use simple string comparison.